packages/vats/test/bootstrapTests/test-vaults-upgrade.js

```diff
@@ -21,8 +21,8 @@ const collateralBrandKey = 'ATOM';
 /**
  * @param {import('ava').ExecutionContext} t
  * @param {object} [options]
- * @param {number} [options.incarnation=1]
- * @param {boolean} [options.logTiming=true]
+ * @param {number} [options.incarnation=1] Default is `1`
+ * @param {boolean} [options.logTiming=true] Default is `true`
```

Comment:
it writes out defaults? interesting

Reply:
I hadn't noticed that. Seems intrusive and redundant to me. I'd expect the doc generator to do that, using the actual default value.

This may be moot with the `no-default-params` lint rule we have. Either way I'll see if I can stop this.